tests/test_quantization/test_quant_args.py

```diff
@@ -82,11 +82,12 @@ def test_actorder():
     with pytest.raises(ValueError):
         QuantizationArgs(strategy="tensor", actorder="weight")
 
-    # test boolean defaulting
+    # test boolean and none defaulting
     assert (
-        QuantizationArgs(group_size=1, actorder="weight").actorder
-        == ActivationOrdering.WEIGHT
+        QuantizationArgs(group_size=1, actorder=True).actorder
+        == ActivationOrdering.GROUP
     )
```

Comment:
Could you keep the weight test as well?

Reply:
A very similar test exists on [line 74](https://github.com/neuralmagic/compressed-tensors/pull/150/files/f333f72ec57d8270649e05faed90639fa376b03e#diff-76bccc3a1a5e7cc3ecb68eceabb4c7eaee15b7ce6eade1b10b888935b8001e30L74-L75)